ide-support/revsaveasiosstandalone.livecodescript

```diff
@@ -924,6 +928,16 @@ private command revCopyMobileFiles pFiles, pBaseFolder, pAppBundle, pAppTarget,
    local tManifest
    set the itemDelimiter to slash
    repeat for each line tFile in pFiles
+      -- Added to support mergExt inclusion for iOS
+      if tFile begins with "!" then
+         local tExtPath
+         put revSBGetExternalPath("ios", char 2 to -1 of tFile) into tExtPath
+         -- if the file prefixed with ! was not a named external, treat normally
+         if tExtPath is not empty then
+            put tExtPath into tFile
+         end if
+      end if
+      
```

Comment:
You've added an almost identical block of code here for iOS and Android for processing "!" externals.  Would it make sense to add a helper function for this?